lib/active_merchant/billing/gateways/decidir_plus.rb

```diff
@@ -82,7 +82,23 @@ def add_purchase_data(post, money, payment, options = {})
         post[:currency] = options[:currency] || self.default_currency
         post[:installments] = options[:installments] || 1
         post[:payment_type] = options[:payment_type] || 'single'
+        add_sub_payments(post, options)
+      end
+
+      def add_sub_payments(post, options)
+        # sub_payments field is required for purchase transactions, even if empty
```

Comment:
Let me know if this comment is unnecessary. I just want people to be aware that this field must be set for transactions against the gateway to be successful.

Reply:
@naashton I'm always down for more comments especially for required fields. I say leave it.